components/layout/Navbar.tsx

```diff
@@ -521,11 +520,11 @@ export default function Navbar(props: NavbarProps) {
             {props.cta[0] === 'checkout' ? (
               <button
                 type="button"
-                onClick={() =>
-                  checkout({
-                    planId: getPlanId(stripeObject),
-                  })
-                }
+                onClick={() => {
+                  // checkout({
```

Comment:
remove this if its unuseful, if not, uncomment the code

Reply:
I leave it like this, for the moment, as it is not used and is only used in campaigns (black friday for example).
`checkout({
                    planId: '',
                  });`